.github/workflows/wasm-bulild-check.yml

```
@@ -0,0 +1,33 @@
+name: CI wasm-check
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  wasm-ci:
+    name: Check wasm build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly
+          components: rustfmt
+      
+      - name: Install wasm-pack
+        run: |
+          curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

We can use `taiki-e/install-action` here.

```suggestion
      - name: Install wasm-pack
        uses: taiki-e/install-action@v2.41.10
        with:
          tool: wasm-pack
          checksum: true
```